onnxruntime/core/providers/dml/DmlExecutionProvider/inc/IWinmlExecutionProvider.h

```diff
@@ -10,6 +10,8 @@
 
 #include "core/framework/op_kernel.h"
 
+#include <DirectML.h>
```

Comment:
Where is this needed?

Reply:
it wasn't needed !

---
In reply to: [354068360](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354068360) [](ancestors = 354068360)